src/middlewared/middlewared/plugins/smb_/utils.py

```diff
@@ -0,0 +1,12 @@
+def smb_strip_comments(auxparam_in):
+        parsed_config = ""
```

Comment:
Is there a reason for the extra large indentation?

Reply:
Yes, I forgot to shift it back by four after moving it to new file :)